src/libraries/shims/System.ValueTuple/ref/System.ValueTuple.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime\ref\System.Runtime.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
GIT shows these additions as new files because more than 50% of the file content has changed (that's a GIT behavior). This is moved from `src/libraries/System.ValueTuple/ref/System.ValueTuple.csproj`. Same for all the other project additions.